components/x-audio/src/redux/index.jsx

```diff
@@ -65,6 +75,23 @@ export default function connectPlayer (Player) {
 			if (this.playingStateAndPropsNeedSync()) {
 				this.updatePlayingStateFromProps(prevProps);
 			}
+
+			if (this.expandedPlayerRef && !this.hasSetSwipeAnimation) {
```

Comment:
Is there any cleanup to do when the component unmounts, `hammer.destroy()` perhaps?

Reply:
👍